mesh/conversation/models.py

```diff
@@ -1,3 +1,6 @@
 from django.db import models
 
 # Create your models here.
+
+class Message(models.Model):
+    pass
```

Comment:
Was this left here by accident, or is it necessary and will be expanded on later? Having a `TODO` here would help clear things up.

Reply:
This was a template for future people